server/services/participants-helper.js

```diff
@@ -73,13 +73,15 @@ const scrubParticipantData = (raw, joinNames) =>
       });
     }
 
+    let rosStatuses = participant.rosStatuses
+      ? participant.rosStatuses.sort((ros1, ros2) => ros2.id - ros1.id)
+      : [];
+    rosStatuses = rosStatuses.filter(rosStatus => sites.includes(rosStatus.rosSite.body.siteId));
```

Comment:
Nice, this won't affect pagination right? We're not affecting the number of records returned, just preventing ROS statuses from sites the requesting user doesn't have access to?

Reply:
Correct!